doc/whats_new/v0.22.rst

```diff
@@ -85,6 +85,11 @@ Changelog
   between `n_jobs=1` and `n_jobs>1` due to the handling of the random state.
   :pr:`9288` by :user:`Bryan Yang <bryanyang0528>`.
 
+- |Fix| Fixed a bug where functions in :mod:`sklearn.cluster._k_means_elkan`
```

Comment:
The class is 
```
:class:`sklearn.cluster.KMeans`
```
, we shouldn't mention private functions in "what's new".

Reply:
Sorry, my bad, fixed.